src/pages/AddResort.js

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+const AddResort = () => {
```

Comment:
- [ ] According to the project requirements for teams of 5 people:

![image](https://github.com/Thinus01/Resort_Booking_Front-end/assets/57421823/2d3c17ce-9931-4f4f-98f7-da5b525b8316)

Since I'm not an admin, the app correctly forbids me to add  resort, however, I can still see and access the links:

![image](https://github.com/Thinus01/Resort_Booking_Front-end/assets/57421823/0120a999-56ae-4382-b40c-db66e9026bef)

Kindly, make those 2 links disappear to nonadmin users.


That way, we can meet this particular requirement. 👍

Reply:
Thanks for your valuable information. I added the necessary instructions inside my project readme.